osu.Game.Rulesets.Taiko/UI/TaikoPlayfield.cs

```diff
@@ -18,169 +15,113 @@
                                 },
-                                HitTarget = new SkinnableDrawable(new TaikoSkinComponent(TaikoSkinComponents.HitTarget), _ => new HitTarget())
+                                HitTarget = new SkinnableDrawable(new TaikoSkinComponent(TaikoSkinComponents.HitTarget), _ => new TaikoHitTarget())
                                 {
-                                    Anchor = Anchor.CentreLeft,
-                                    Origin = Anchor.Centre,
                                     RelativeSizeAxes = Axes.Both,
-                                    FillMode = FillMode.Fit
                                 }
                             }
                         },
-                        barlineContainer = new ProxyContainer
-                        {
-                            RelativeSizeAxes = Axes.Both,
-                            Padding = new MarginPadding { Left = HIT_TARGET_OFFSET }
-                        },
-                        new Container
-                        {
-                            Name = "Hit objects",
-                            RelativeSizeAxes = Axes.Both,
-                            Padding = new MarginPadding { Left = HIT_TARGET_OFFSET },
-                            Masking = true,
-                            Child = HitObjectContainer
-                        },
-                        kiaiExplosionContainer = new Container<KiaiHitExplosion>
+                        hitTargetOffsetContent = new Container
                         {
-                            Name = "Kiai hit explosions",
                             RelativeSizeAxes = Axes.Both,
-                            FillMode = FillMode.Fit,
-                            Margin = new MarginPadding { Left = HIT_TARGET_OFFSET },
-                            Blending = BlendingParameters.Additive
-                        },
-                        judgementContainer = new JudgementContainer<DrawableTaikoJudgement>
-                        {
-                            Name = "Judgements",
-                            RelativeSizeAxes = Axes.Y,
-                            Margin = new MarginPadding { Left = HIT_TARGET_OFFSET },
-                            Blending = BlendingParameters.Additive
+                            Children = new Drawable[]
+                            {
+                                barlineContainer = new ProxyContainer
+                                {
+                                    RelativeSizeAxes = Axes.Both,
+                                },
+                                new Container
+                                {
+                                    Name = "Hit objects",
+                                    RelativeSizeAxes = Axes.Both,
+                                    Child = HitObjectContainer
+                                },
+                                kiaiExplosionContainer = new Container<KiaiHitExplosion>
```

Comment:
This should not be masked. The kiai glows are supposed to exceed the playfield:

![image](https://user-images.githubusercontent.com/1329837/80172737-be6f4b00-8628-11ea-8536-eac9d44a3dc8.png)

![image](https://user-images.githubusercontent.com/1329837/80172743-c3cc9580-8628-11ea-9e25-2d8325e431ff.png)